I can not build application

lindol@lindol-laptop:~/myjobs/StickFace/StickFace$ gcc src.c $(pkg-config --cflags gtk+-3.0) -o sticky
Package gtk+-3.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gtk+-3.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gtk+-3.0' found
src.c:11:21: fatal error: gtk/gtk.h: 그런 파일이나 디렉터리가 없습니다
 #include <gtk/gtk.h>
                     ^
compilation terminated.
lindol@lindol-laptop:~/myjobs/StickFace/StickFace$ 

---

I just following your compile guide
but i read this link " http://askubuntu.com/questions/101306/how-do-i-install-gtk-3-0"

you need to "how to install requirement package for a compile" to compile section. 

so...
After installed libgtk-3-dev package.
but i still have problem below output.

lindol@lindol-laptop:~/myjobs/StickFace/StickFace$ gcc src.c $(pkg-config --cflags gtk+-3.0) -o sticky
src.c: In function ‘main_screen’:
src.c:197:3: warning: ‘gtk_window_set_has_resize_grip’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtkwindow.h:472) [-Wdeprecated-declarations]
   gtk_window_set_has_resize_grip(GTK_WINDOW(main.window),TRUE);
   ^
src.c:294:3: warning: ‘gtk_tool_button_new_from_stock’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtktoolbutton.h:84): Use 'gtk_tool_button_new' instead [-Wdeprecated-declarations]
   main.button_second_first = gtk_tool_button_new_from_stock(GTK_STOCK_NEW);
   ^
src.c:294:3: warning: ‘GtkStock’ is deprecated [-Wdeprecated-declarations]
src.c:296:3: warning: ‘gtk_tool_button_new_from_stock’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtktoolbutton.h:84): Use 'gtk_tool_button_new' instead [-Wdeprecated-declarations]
   main.button_second_second = gtk_tool_button_new_from_stock(GTK_STOCK_ABOUT);
   ^
src.c:296:3: warning: ‘GtkStock’ is deprecated [-Wdeprecated-declarations]
src.c: In function ‘main_screen_from_data’:
src.c:427:3: warning: ‘gtk_window_set_has_resize_grip’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtkwindow.h:472) [-Wdeprecated-declarations]
   gtk_window_set_has_resize_grip(GTK_WINDOW(main[j].window),TRUE);
   ^
src.c:524:3: warning: ‘gtk_tool_button_new_from_stock’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtktoolbutton.h:84): Use 'gtk_tool_button_new' instead [-Wdeprecated-declarations]
   main[j].button_second_first = gtk_tool_button_new_from_stock(GTK_STOCK_NEW);
   ^
src.c:524:3: warning: ‘GtkStock’ is deprecated [-Wdeprecated-declarations]
src.c:526:3: warning: ‘gtk_tool_button_new_from_stock’ is deprecated (declared at /usr/include/gtk-3.0/gtk/gtktoolbutton.h:84): Use 'gtk_tool_button_new' instead [-Wdeprecated-declarations]
   main[j].button_second_second = gtk_tool_button_new_from_stock(GTK_STOCK_ABOUT);
   ^
src.c:526:3: warning: ‘GtkStock’ is deprecated [-Wdeprecated-declarations]
/tmp/ccoWGJtD.o: In function `on_button_press':
src.c:(.text+0x69): undefined reference to`gtk_window_get_type'
src.c:(.text+0x78): undefined reference to `gtk_widget_get_toplevel'
src.c:(.text+0x83): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x96): undefined reference to `gtk_window_begin_move_drag'
/tmp/ccoWGJtD.o: In function`toggle_label':
src.c:(.text+0xc6): undefined reference to `gtk_tool_button_get_label'
src.c:(.text+0xf0): undefined reference to`gtk_tool_button_set_label'
src.c:(.text+0xf5): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x107): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x114): undefined reference to `gtk_tool_item_set_tooltip_text'
src.c:(.text+0x127): undefined reference to`gtk_tool_button_set_label'
src.c:(.text+0x12c): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x13e): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x14b): undefined reference to `gtk_tool_item_set_tooltip_text'
/tmp/ccoWGJtD.o: In function`show_window_bar':
src.c:(.text+0x16a): undefined reference to `gtk_window_set_decorated'
/tmp/ccoWGJtD.o: In function`hide_window_bar':
src.c:(.text+0x189): undefined reference to `gtk_window_set_decorated'
/tmp/ccoWGJtD.o: In function`hide':
src.c:(.text+0x1b6): undefined reference to `gtk_window_get_type'
src.c:(.text+0x1cb): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1d8): undefined reference to `gtk_window_set_keep_above'
src.c:(.text+0x1dd): undefined reference to`gtk_window_get_type'
src.c:(.text+0x1f2): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1ff): undefined reference to`gtk_window_set_keep_below'
src.c:(.text+0x21c): undefined reference to `g_print'
src.c:(.text+0x223): undefined reference to`gtk_window_get_type'
src.c:(.text+0x238): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x245): undefined reference to`gtk_window_set_keep_below'
src.c:(.text+0x24a): undefined reference to `gtk_window_get_type'
src.c:(.text+0x25f): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x26c): undefined reference to `gtk_window_set_keep_above'
src.c:(.text+0x289): undefined reference to`g_print'
/tmp/ccoWGJtD.o: In function `about':
src.c:(.text+0x2a0): undefined reference to`gtk_about_dialog_new'
src.c:(.text+0x2b3): undefined reference to `g_print'
src.c:(.text+0x2b8): undefined reference to`gtk_about_dialog_get_type'
src.c:(.text+0x2ca): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x2d7): undefined reference to`gtk_about_dialog_set_program_name'
src.c:(.text+0x2dc): undefined reference to `gtk_about_dialog_get_type'
src.c:(.text+0x2ee): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x2fb): undefined reference to `gtk_about_dialog_set_version'
src.c:(.text+0x300): undefined reference to`gtk_about_dialog_get_type'
src.c:(.text+0x312): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x31f): undefined reference to`gtk_about_dialog_set_comments'
src.c:(.text+0x324): undefined reference to `gtk_about_dialog_get_type'
src.c:(.text+0x336): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x343): undefined reference to `gtk_about_dialog_set_website'
src.c:(.text+0x348): undefined reference to`gtk_about_dialog_get_type'
src.c:(.text+0x35a): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x367): undefined reference to`gtk_about_dialog_set_copyright'
src.c:(.text+0x36c): undefined reference to `gtk_dialog_get_type'
src.c:(.text+0x37e): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x386): undefined reference to `gtk_dialog_run'
src.c:(.text+0x392): undefined reference to`gtk_widget_destroy'
/tmp/ccoWGJtD.o: In function `dynamic_size_shift':
src.c:(.text+0x402): undefined reference to`gtk_window_set_title'
/tmp/ccoWGJtD.o: In function `main_screen':
src.c:(.text+0x44b): undefined reference to`gtk_window_new'
src.c:(.text+0x457): undefined reference to `gtk_window_get_type'
src.c:(.text+0x46c): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x47e): undefined reference to `gtk_window_set_default_size'
src.c:(.text+0x483): undefined reference to`gtk_window_get_type'
src.c:(.text+0x498): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x4a5): undefined reference to`gtk_window_set_title'
src.c:(.text+0x4aa): undefined reference to `gtk_window_get_type'
src.c:(.text+0x4bf): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x4cc): undefined reference to `gtk_window_set_decorated'
src.c:(.text+0x4e0): undefined reference to`gtk_widget_add_events'
src.c:(.text+0x4e5): undefined reference to `gtk_window_get_type'
src.c:(.text+0x4fa): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x507): undefined reference to `gtk_window_set_keep_above'
src.c:(.text+0x50c): undefined reference to`gtk_window_get_type'
src.c:(.text+0x521): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x52e): undefined reference to`gtk_window_set_has_resize_grip'
src.c:(.text+0x533): undefined reference to `gtk_window_get_type'
src.c:(.text+0x548): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x555): undefined reference to `gtk_window_set_resizable'
src.c:(.text+0x561): undefined reference to`gtk_text_view_new'
src.c:(.text+0x57b): undefined reference to `gtk_widget_set_size_request'
src.c:(.text+0x580): undefined reference to`gtk_widget_get_type'
src.c:(.text+0x592): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x59f): undefined reference to`gtk_widget_set_vexpand'
src.c:(.text+0x5a4): undefined reference to `gtk_widget_get_type'
src.c:(.text+0x5b6): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x5c3): undefined reference to `gtk_widget_set_hexpand'
src.c:(.text+0x5c8): undefined reference to`gtk_text_view_get_type'
src.c:(.text+0x5da): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x5e7): undefined reference to`gtk_text_view_set_wrap_mode'
src.c:(.text+0x5f6): undefined reference to `gtk_scrolled_window_new'
src.c:(.text+0x610): undefined reference to`gtk_widget_set_size_request'
src.c:(.text+0x619): undefined reference to `gtk_container_get_type'
src.c:(.text+0x62b): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x636): undefined reference to `gtk_container_add'
src.c:(.text+0x63b): undefined reference to`gtk_grid_new'
src.c:(.text+0x64e): undefined reference to `gtk_container_get_type'
src.c:(.text+0x663): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x66e): undefined reference to `gtk_container_add'
src.c:(.text+0x673): undefined reference to`gtk_button_new'
src.c:(.text+0x689): undefined reference to `gtk_tool_button_new'
src.c:(.text+0x692): undefined reference to`gtk_tool_item_get_type'
src.c:(.text+0x6a4): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x6b1): undefined reference to`gtk_tool_item_set_tooltip_text'
src.c:(.text+0x6c0): undefined reference to `gtk_tool_button_new'
src.c:(.text+0x6ce): undefined reference to`gtk_tool_button_new_from_stock'
src.c:(.text+0x6d7): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x6e9): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x6f6): undefined reference to `gtk_tool_item_set_tooltip_text'
src.c:(.text+0x700): undefined reference to`gtk_tool_button_new_from_stock'
src.c:(.text+0x709): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x71b): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x728): undefined reference to `gtk_tool_item_set_tooltip_text'
src.c:(.text+0x72d): undefined reference to`gtk_toolbar_new'
src.c:(.text+0x736): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x748): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x755): undefined reference to `gtk_toolbar_set_style'
src.c:(.text+0x75a): undefined reference to`gtk_container_get_type'
src.c:(.text+0x76c): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x779): undefined reference to`gtk_container_set_border_width'
src.c:(.text+0x77e): undefined reference to `gtk_toolbar_new'
src.c:(.text+0x787): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x799): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x7a6): undefined reference to`gtk_toolbar_set_style'
src.c:(.text+0x7ab): undefined reference to `gtk_container_get_type'
src.c:(.text+0x7bd): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x7ca): undefined reference to `gtk_container_set_border_width'
src.c:(.text+0x7d3): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x7e5): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x7f5): undefined reference to`gtk_toolbar_insert'
src.c:(.text+0x7fe): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x810): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x820): undefined reference to `gtk_toolbar_insert'
src.c:(.text+0x829): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x83b): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x84b): undefined reference to`gtk_toolbar_insert'
src.c:(.text+0x854): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x866): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x876): undefined reference to `gtk_toolbar_insert'
src.c:(.text+0x87f): undefined reference to`gtk_grid_get_type'
src.c:(.text+0x894): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x8b5): undefined reference to`gtk_grid_attach'
src.c:(.text+0x8be): undefined reference to `gtk_grid_get_type'
src.c:(.text+0x8d3): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x8f4): undefined reference to `gtk_grid_attach'
src.c:(.text+0x8fd): undefined reference to`gtk_grid_get_type'
src.c:(.text+0x912): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x933): undefined reference to`gtk_grid_attach'
src.c:(.text+0x947): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x96a): undefined reference to`g_signal_connect_data'
src.c:(.text+0x97e): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x99c): undefined reference to`gtk_main_quit'
src.c:(.text+0x9a9): undefined reference to `g_signal_connect_data'
src.c:(.text+0x9ba): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x9e2): undefined reference to `g_signal_connect_data'
src.c:(.text+0x9f7): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xa18): undefined reference to `g_signal_connect_data'
src.c:(.text+0xa29): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xa3f): undefined reference to `gtk_main_quit'
src.c:(.text+0xa4c): undefined reference to`g_signal_connect_data'
src.c:(.text+0xa5d): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xa80): undefined reference to`g_signal_connect_data'
src.c:(.text+0xa91): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xab4): undefined reference to`g_signal_connect_data'
src.c:(.text+0xacf): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xaf0): undefined reference to`g_signal_connect_data'
src.c:(.text+0xb0b): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xb2c): undefined reference to`g_signal_connect_data'
src.c:(.text+0xb47): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xb68): undefined reference to`g_signal_connect_data'
src.c:(.text+0xb77): undefined reference to `gtk_widget_show_all'
src.c:(.text+0xb7c): undefined reference to`gtk_main'
/tmp/ccoWGJtD.o: In function `main_screen_from_data':
src.c:(.text+0xcf0): undefined reference to`gtk_window_new'
src.c:(.text+0xd1d): undefined reference to `gtk_window_get_type'
src.c:(.text+0xd4d): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xd5f): undefined reference to `gtk_window_set_default_size'
src.c:(.text+0xd64): undefined reference to`gtk_window_get_type'
src.c:(.text+0xd94): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xda1): undefined reference to`gtk_window_set_title'
src.c:(.text+0xda6): undefined reference to `gtk_window_get_type'
src.c:(.text+0xdd6): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xde3): undefined reference to `gtk_window_set_decorated'
src.c:(.text+0xe15): undefined reference to`gtk_widget_add_events'
src.c:(.text+0xe1a): undefined reference to `gtk_window_get_type'
src.c:(.text+0xe4a): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xe57): undefined reference to `gtk_window_set_keep_above'
src.c:(.text+0xe5c): undefined reference to`gtk_window_get_type'
src.c:(.text+0xe8c): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0xe99): undefined reference to`gtk_window_set_has_resize_grip'
src.c:(.text+0xe9e): undefined reference to `gtk_window_get_type'
src.c:(.text+0xece): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xedb): undefined reference to `gtk_window_set_resizable'
src.c:(.text+0xf0e): undefined reference to`gtk_text_view_new'
src.c:(.text+0xf3f): undefined reference to `gtk_text_view_get_type'
src.c:(.text+0xf73): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0xf7b): undefined reference to `gtk_text_view_get_buffer'
src.c:(.text+0x1016): undefined reference to`gtk_text_buffer_set_text'
src.c:(.text+0x1051): undefined reference to `gtk_widget_set_size_request'
src.c:(.text+0x1056): undefined reference to`gtk_widget_get_type'
src.c:(.text+0x108a): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1097): undefined reference to`gtk_widget_set_vexpand'
src.c:(.text+0x109c): undefined reference to `gtk_widget_get_type'
src.c:(.text+0x10d0): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x10dd): undefined reference to `gtk_widget_set_hexpand'
src.c:(.text+0x10e2): undefined reference to`gtk_text_view_get_type'
src.c:(.text+0x1116): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1123): undefined reference to`gtk_text_view_set_wrap_mode'
src.c:(.text+0x1132): undefined reference to `gtk_scrolled_window_new'
src.c:(.text+0x1199): undefined reference to`gtk_widget_set_size_request'
src.c:(.text+0x11c7): undefined reference to `gtk_container_get_type'
src.c:(.text+0x11fb): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1206): undefined reference to `gtk_container_add'
src.c:(.text+0x120b): undefined reference to`gtk_grid_new'
src.c:(.text+0x1265): undefined reference to `gtk_container_get_type'
src.c:(.text+0x1295): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x12a0): undefined reference to `gtk_container_add'
src.c:(.text+0x12a5): undefined reference to`gtk_button_new'
src.c:(.text+0x12dd): undefined reference to `gtk_tool_button_new'
src.c:(.text+0x130f): undefined reference to`gtk_tool_item_get_type'
src.c:(.text+0x1344): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1351): undefined reference to`gtk_tool_item_set_tooltip_text'
src.c:(.text+0x1360): undefined reference to `gtk_tool_button_new'
src.c:(.text+0x1396): undefined reference to`gtk_tool_button_new_from_stock'
src.c:(.text+0x13c8): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x13fd): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x140a): undefined reference to `gtk_tool_item_set_tooltip_text'
src.c:(.text+0x1414): undefined reference to`gtk_tool_button_new_from_stock'
src.c:(.text+0x1445): undefined reference to `gtk_tool_item_get_type'
src.c:(.text+0x1479): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1486): undefined reference to `gtk_tool_item_set_tooltip_text'
src.c:(.text+0x148b): undefined reference to`gtk_toolbar_new'
src.c:(.text+0x14bd): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x14f2): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x14ff): undefined reference to `gtk_toolbar_set_style'
src.c:(.text+0x1504): undefined reference to`gtk_container_get_type'
src.c:(.text+0x1539): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1546): undefined reference to`gtk_container_set_border_width'
src.c:(.text+0x154b): undefined reference to `gtk_toolbar_new'
src.c:(.text+0x157d): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x15b2): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x15bf): undefined reference to`gtk_toolbar_set_style'
src.c:(.text+0x15c4): undefined reference to `gtk_container_get_type'
src.c:(.text+0x15f9): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1606): undefined reference to `gtk_container_set_border_width'
src.c:(.text+0x1635): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x166a): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x167a): undefined reference to`gtk_toolbar_insert'
src.c:(.text+0x16a8): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x16dd): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x16ed): undefined reference to `gtk_toolbar_insert'
src.c:(.text+0x171c): undefined reference to`gtk_toolbar_get_type'
src.c:(.text+0x1751): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1761): undefined reference to`gtk_toolbar_insert'
src.c:(.text+0x178f): undefined reference to `gtk_toolbar_get_type'
src.c:(.text+0x17c4): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x17d4): undefined reference to `gtk_toolbar_insert'
src.c:(.text+0x1803): undefined reference to`gtk_grid_get_type'
src.c:(.text+0x1837): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1858): undefined reference to`gtk_grid_attach'
src.c:(.text+0x1886): undefined reference to `gtk_grid_get_type'
src.c:(.text+0x18ba): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x18db): undefined reference to `gtk_grid_attach'
src.c:(.text+0x190a): undefined reference to`gtk_grid_get_type'
src.c:(.text+0x193e): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x195f): undefined reference to`gtk_grid_attach'
src.c:(.text+0x1991): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x19b4): undefined reference to`g_signal_connect_data'
src.c:(.text+0x19e6): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1a22): undefined reference to`gtk_main_quit'
src.c:(.text+0x1a2f): undefined reference to `g_signal_connect_data'
src.c:(.text+0x1a66): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1a8b): undefined reference to `g_signal_connect_data'
src.c:(.text+0x1aec): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1b0d): undefined reference to `g_signal_connect_data'
src.c:(.text+0x1b43): undefined reference to`g_type_check_instance_cast'
src.c:(.text+0x1b59): undefined reference to `gtk_main_quit'
src.c:(.text+0x1b66): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1b9d): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1bc0): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1bf6): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1c19): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1c70): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1c91): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1ce8): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1d09): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1d60): undefined reference to `g_type_check_instance_cast'
src.c:(.text+0x1d81): undefined reference to`g_signal_connect_data'
src.c:(.text+0x1dae): undefined reference to `gtk_widget_show_all'
src.c:(.text+0x1dc3): undefined reference to`gtk_main'
/tmp/ccoWGJtD.o: In function `main':
src.c:(.text+0x1e2d): undefined reference to`gtk_init'
collect2: error: ld returned 1 exit status

---

please let me know your compile tip. :)
I am waiting for your help.
Thank you

This is my system.

lindol@lindol-laptop:~/myjobs/StickFace/StickFace$ gnome-shell --version
GNOME Shell 3.14.3
lindol@lindol-laptop:~/myjobs/StickFace/StickFace$ uname -a
Linux lindol-laptop 3.16.0-28-generic #38-Ubuntu SMP Fri Dec 12 17:37:40 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

My system installed Ubuntu GNOME 14.10 (64bit)

--- a/README.md
+++ b/README.md
@@ -1,34 +1,46 @@
 # StickFace
-An Awesome DeskTop Sticky Widget which Follows you Everywhere .Written in C language.Built With Gtk For Gnome and Unity Environment.Well Tested Under Ubuntu
+An awesome desktop sticky widget which follows you everywhere. Written in C and built with GTK for GNOME and Unity desktop environments. Well tested under Ubuntu.
 
-#Requirements
+# Requirements
 1. Linux
 2. GCC
 3. Gtk-Bundle
 
-#Compiling The src.c
+# Install dependencies
+## Ubuntu
+```shell
+sudo apt-get install gcc libgtk-3-dev
+```
+
+# Compiling from source
 ```shell
 gcc src.c $(pkg-config --cflags --libs gtk+-3.0) -o sticky
 ```
-#Features
-1. StickFace's special feature is to follow you around your WorkSpace Until You Give it Command "Go To DeskTop"
-![Go To DeskTop](https://www.anonimg.com/img/910bd31e0b3fd9e75cf5b8c9ce7eef65.png "Go To DeskTop")
-2. Click on "Follow Me" Again Toggles StickFace To Follow You Around The Screen 
+# Features
+1. StickFace's special feature is to follow you around your workspace until you give it the command "Go To Desktop"
+
+![Go To Desktop](https://www.anonimg.com/img/910bd31e0b3fd9e75cf5b8c9ce7eef65.png "Go to Desktop")
+
+2. Click on "Follow Me" again toggles StickFace to follow you around the screen 
+
 ![Follow Me](https://www.anonimg.com/img/198aa7304b87c25d5318c55fcb145ae7.png "Follow Me")
-3. Click on "Add New" Creates a New Stick Screen For Second Note
+
+3. Click on "Add New" creates a New Stick Screen for a second note
+
 ![New](https://www.anonimg.com/img/a604405c31d8b68c327b06633e6e498b.png "New Note")
 
-#Usage
-> 1. data_file_1.stick (is the Database File it is resonsible for holding databse for StickFace)
-> 2. Run the Executable 
+# Usage
+1. data_file_1.stick (is the database file and is responsible for holding the database for StickFace)
+2. Run the executable: `sticky`
 
-#Contributing
-I'll Try To Merge Every Pull Request Which I'll Get
+# Contributing
+I'll try to merge every pull request which I receive.
 
-#Bugs and Errors
-#####There are Lot Of Bugs in StickFace It is Not Stable For Use ,Help Me Fix Them
-1. Every Time StickFace Loads Data From Datafile 4 Extra StickFace Comes into Runtime Stack
-2. DataBase Update Module Not Ready
-3. Glitches Not Fixed
+# Bugs and Errors
+**There are a few bugs in StickFace. It is not stable for daily use yet. Help me fix them!**
+
+1. Every time StickFace loads data from the data file, four extra StickFace interfaces comes into the run-time stack
+2. Database update module not ready
+3. Glitches not fixed
 4. Some Times Become Annoying ( UX problem)
 

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ gcc src.c $(pkg-config --cflags --libs gtk+-3.0) -o sticky
 2. Run the executable: `sticky`
 
 # Contributing
-I'll try to merge every pull request which I receive.
+I'll try to merge every pull request that I receive.
 
 # Bugs and Errors
 **There are a few bugs in StickFace. It is not stable for daily use yet. Help me fix them!**
@@ -42,5 +42,5 @@ I'll try to merge every pull request which I receive.
 1. Every time StickFace loads data from the data file, four extra StickFace interfaces comes into the run-time stack
 2. Database update module not ready
 3. Glitches not fixed
-4. Some Times Become Annoying ( UX problem)
+4. Some times it becomes annoying (UX problem)
 

